src/Symfony/Component/Workflow/Definition.php

```diff
@@ -75,23 +90,27 @@ public function getMetadataStore(): MetadataStoreInterface
         return $this->metadataStore;
     }
 
-    private function setInitialPlace(string $place = null)
+    private function setInitialPlaces($places = null)
     {
-        if (null === $place) {
+        if (null === $places) {
             return;
         }
 
-        if (!isset($this->places[$place])) {
-            throw new LogicException(sprintf('Place "%s" cannot be the initial place as it does not exist.', $place));
+        $places = (array) $places;
+
+        foreach ($places as $place) {
+            if (!isset($this->places[$place])) {
+                throw new LogicException(sprintf('Place "%s" cannot be the initial place as it does not exist.', $place));
```

Comment:
probably a better place for an error if marking store is `single_state` and we do have more than one initial place ?

Reply:
This is absolutely not related.